pkgs/by-name/hy/hyprlang/package.nix

```diff
@@ -1,30 +1,32 @@
-{
-  lib,
-  stdenv,
-  fetchFromGitHub,
-  cmake,
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
 }:
-stdenv.mkDerivation (finalAttrs: {
+
+stdenv.mkDerivation rec {
```

Comment:
Why revert to `rec`?

Reply:
nix-init did this
I used it for both, didn't realize that hyprlang had a package alr